src/client/components/dealtcard/dealtcard.js

```diff
@@ -4,12 +4,13 @@ import PropTypes from 'prop-types';
 class DealtCard extends React.Component {
   static propTypes = {
     card: PropTypes.string.isRequired,
+    gamemode: PropTypes.string.isRequired,
   };
 
   render() {
     const cardClass = this.props.card ? this.props.card.toLowerCase() : '-none';
     return (
-      <div className={`playing-card card${cardClass} active card-rounded scaled-big`} />
+      <div className={`playing-card ${(this.props.gamemode === "Cornucopia") ? "c" : ""}card${this.props.card.toLowerCase()} active card-rounded scaled-big`} />
```

Comment:
Very often you are using gameMode === Cornucopia. Could you move this to a dedicated function like isGameModeCornucopia?